### DISCUSSION TITLE
Is there a way to set the uid of the cy.callFirestore call ?

### DISCUSSION THREAD

**Discussion by Tockra** [prescottprue/cypress-firebase#1002] [General]
Hi,

I want to check my firebase rules. 
For that I want to do some requests to my firebase emulator. 
But currently the requests have all access, because the service admin account is used.
Is there a way to set a UID for the specific call?

T

**Comment by prescottprue** [comment]
There is not currently since calls happen through the backend (intentionally to be able to verify changes made by UI). This library is mostly to test results of changes made by a UI instead of rules changes

Might be worth checking out the Firebase docs on testing rules: https://firebase.google.com/docs/firestore/security/test-rules-emulator
